isis/tests/FunctionalTestsIsisImportEis.cpp

```diff
@@ -83,10 +86,94 @@ TEST_F(TempTestingFiles, FunctionalTestIsisImportEisNacFrame){
     End_Object
     End
   )");
-  QString dataFileName = " data/eis2isis/nacFrame/nac000xxx_2022145t000000_0000000001_frame_raw02.xml";
+  QString dataFilePath= " data/isisimport/eispds/nacFrame/nac000xxx_2022145t000000_0000000001_frame_raw02.xml";
+  QString dataFileName = "nac000xxx_2022145t000000_0000000001_frame_raw02.xml";
+  QString imageFilePath = " data/isisimport/eispds/nacFrame/nac000xxx_2022145t000000_0000000001_frame_raw02.img";
+  QString imageFileName = "nac000xxx_2022145t000000_0000000001_frame_raw02.img";
   QString templateFile = "../appdata/import/PDS4/ClipperEIS.tpl";
   QString cubeFileName = tempDir.path() + "/nacFrame.cub";
-  QVector<QString> args = {"from=" + dataFileName, "to=" + cubeFileName, "template=" + templateFile};
+
+  // hard coding these values for now, but we should use some sort of xml parser to grab these values from the labels xml
```

Comment:
I don't know if this needs to be dynamically parsed out of the XML. We also have them hard coded into the truth label.